appinfo/routes.php

```diff
@@ -94,6 +95,11 @@
 			'url' => '/api/accounts/{id}/signature',
 			'verb' => 'PUT'
 		],
+		[
+			'name' => 'accounts#updateSmimeCertificate',
+			'url' => '/api/accounts/{id}/smimeCertificate',
```

Comment:
nit: smime-certificate? camel case in an URL feels a bit strange

Reply:
Done